src/main/java/org/craftercms/studio/api/v2/repository/RepositoryChanges.java

```diff
@@ -19,6 +19,7 @@
 
 import org.apache.commons.collections4.CollectionUtils;
 
+import java.util.HashSet;
```

Comment:
Is this the right data structure to use? Do we have concerns about the size of this set in memory? Should we consider TreeSet.

Reply:
As discussed, we are changing this to use ArrayList by default, since we the uniqueness should already be guaranteed by the updated/deleted path Sets